### PR TITLE
Renamed a few attributes in SMS/reminders

### PR DESCRIPTION
**Pull request by orangejenny**
https://github.com/dimagi/commcare-hq/pull/25757#pullrequestreview-309283197

Looked safe to do this and drop a few other `cchq` prefixes.

### CODE REVIEW

**Review by esoergel — APPROVED** [review]
This isn't some kind of namespacing thing is it?  Like non-private attributes being rendered in the form by default or something?

**Review by millerdev — APPROVED** [review]
I wonder why those attributes were "private" in the first place? I assume it doesn't have something to do with JsonObject persistence? (all non-private attributes are automatically included in the result of `obj.to_json()`)

**Comment by orangejenny** [comment]
I think it's just a convention. I opened up the `KeywordForm` and `SettingsForm` locally and saved values, and all seemed to be well. I don't think form classes would be likely to use `to_json`.

File: corehq/apps/reminders/forms.py [hunk 1]
```diff
@@ -128,8 +128,8 @@ def clean(self, value):
 
 
 class KeywordForm(Form):
-    _cchq_domain = None
-    _sk_id = None
```
Comment (mkangia) [inline corehq/apps/reminders/forms.py]: I would have interpreted that as `simon kelly id`

Comment (orangejenny) [reply corehq/apps/reminders/forms.py]: I hope @snopoke is flattered.

**Pull request merged**